tests/frontend/test_circuitspecs_chip2.py

```diff
@@ -367,6 +395,62 @@ def test_no_unitary(self, tol):
 
         assert program_equivalence(res, expected, atol=tol)
 
+        # double check that the applied symplectic is correct
+
+        # remove the Fock measurements
+        res.circuit = res.circuit[:-1]
+
+        # extract the Gaussian symplectic matrix
+        O = res.compile("gaussian_unitary").circuit[0].op.p[0]
+
+        # construct the expected symplectic matrix corresponding
+        # to just the initial two mode squeeze gates
+        S = TMS(SQ_AMPLITUDE, 0)
+
+        expected = np.zeros([2*8, 2*8])
+        idx = np.arange(2*8).reshape(4, 4).T
+        for i in idx:
+            expected[i.reshape(-1, 1), i.reshape(1, -1)] = S
+
+        assert np.allclose(O, expected, atol=tol)
+
+    def test_mz_gate(self, tol):
```

Comment:
@nquesada can you have a look at this test please?

Reply:
@nquesada, once you've given the ✔️ on this test, I will merge in this PR :)